FreeRTOS-Plus/Demo/FreeRTOS_Plus_WolfSSL_FIPS_Ready_Windows_Simulator/README_wolfSSL_FIPS_ready.md

```diff
@@ -0,0 +1,91 @@
+
+# How to build and run the Demo application
+
+By double-clicking the solution file named "FreeRTOS_Plus_WolfSSL_FIPS_Ready.sln" included in this folder, Visual Studio starts and shows you a project in its solution explorer. It is named "RTOSDemo" and provides a console application program which runs on windows. 
+
+All required settings have been set in the user_settings.h header file included in the RTOSDemo/FreeRTOS+/wolfSSL folder in the solution explorer pane. The next step is to build the RTOSDemo application:
+
+1. Build the RTOSDemo project
+2. Run the RTOSDemo.exe 
+
+You will see a console that pops up, and it shows output like the following:
+
+```
+Starting Power On Self Test
+In core integrity check error: hash = C66491A040D5B9686BAA7A75A280290D91B49...
+ERROR: -203
+Power On Self Test(Known-Answer-Test) FAILURE
+```
+
+Do not warry about this result, an error is expected at this point. Error number "-203" means In-Core-integrity-check failed. The check is identical to the "In Core Integrity Test" listed in the previous section. And the subsequent KAT also failed due to the first error. Once FIPS Ready has failed POST, it enters an error state and never allows subsequent cryptographic operations until the device is restarted and the tests can complete successfully. 
+
+The in-core integrity check requires a pre-calclated hash value to be stored in the fips_test.c source file. Remember that you did not yet set this pre-calculated value durling the build process. Because the hash does not match the stored value is the reason why this first run will fail.
+
+# Update Pre-calculated hash value
+
+1. Let us go back to the messages in the console shown in the previous section. You may see "hash = C66491A040..." in the message. **The charactor sequence is the value for the pre-calculated hash value.** Please copy this charactor sequece and store it in a temporary location for reference in the next step.
+
+2. Find and open the file named "**fips_test.c**" under the RTOSDemo/FreeRTOS+/wolfSSL/wolfcrypt folder in the Visual Studio solution explorer pane. Or you can reach the file by traversing "../../Source/WolfSSL-FIPS-Ready/wolfcrypt/src/fips_test.c" starting from the folder where this README is included.
+
+3. In the fips_test.c, find the following statement: 
+
+    ```
+    static const char verifyCore[] =
+        "903B291C50C8F0BAB8D2C632853C6D827A9022A4B12260C3A14F4BEBD101228";
+    ```
+
+    Replace "903b291C..." with the character sequecece(C66491A040... ) you have stored in your temporary location from above. Save fips_test.c and build the application.
+
+4. Run the application.
+
+    This time, you should see:
+
+    ```
+    Starting Power On Self Test
+    Power On Self Test SUCCESS
+    Waiting for new connection
+    Connection established

```

Comment:
What server are we connecting too? BTW this README was really easy to follow, nice work :)